gemrb/core/Interface.cpp

```diff
@@ -3077,6 +3091,9 @@ void Interface::DrawWindows(bool allow_delete)
 				}
 			} else if (win->Visible) {
 				win->DrawWindow();
+			} else if (!win->Visible && (t <= topwin.size() - 2) && !backgroundRedrawn) {
```

Comment:
where does the -2 come from?

Reply:
I remember having a top-window in IWD2 that was invisble: the loading screen. Which then ultimately blocked the game screen. I checked again, _that_ doesn't occur with `backgroundRedrawn ` any more as well. The latter one was only to draw that once, for performance, but I'm not sure if this really cancels out other such cases.

Reply:
Well, if it is still needed, it deserves a comment. Even comparing t to topwin size looks odd considering what it holds.

By using < instead of <= the constant can be lowered to -1, but that still sticks out then.

Reply:
Ok, I will look into it.